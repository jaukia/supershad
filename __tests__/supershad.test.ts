import supershad from '../src/index';

test("supershad", () => {
  expect(supershad()).toMatchSnapshot();
});