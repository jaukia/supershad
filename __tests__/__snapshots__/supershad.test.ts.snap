// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`supershad 1`] = `
Array [
  "0.08px 0.26px 0.47px -0.02px hsl(var(--shadow-color) / 0.14),
0.29px 0.95px 1.57px -0.07px hsl(var(--shadow-color) / 0.13),
0.88px 2.89px 7.02px -0.3px hsl(var(--shadow-color) / 0.12),
0px 0px 0px 1px hsl(var(--shadow-color) / 0.04)",
  "0.2px 0.67px 0.73px -0.05px hsl(var(--shadow-color) / 0.12),
0.65px 2.12px 2.01px -0.15px hsl(var(--shadow-color) / 0.11),
1.53px 5.01px 4.85px -0.36px hsl(var(--shadow-color) / 0.1),
3.53px 11.55px 16.07px -1.21px hsl(var(--shadow-color) / 0.09),
0px 0px 0px 1px hsl(var(--shadow-color) / 0.04)",
  "0.23px 0.77px 0.79px -0.07px hsl(var(--shadow-color) / 0.08),
0.63px 2.05px 1.84px -0.16px hsl(var(--shadow-color) / 0.07),
1.22px 3.99px 3.31px -0.29px hsl(var(--shadow-color) / 0.07),
2.1px 6.87px 5.49px -0.48px hsl(var(--shadow-color) / 0.07),
3.4px 11.14px 9.04px -0.8px hsl(var(--shadow-color) / 0.06),
5.44px 17.82px 15.8px -1.4px hsl(var(--shadow-color) / 0.06),
8.82px 28.87px 34.18px -3.02px hsl(var(--shadow-color) / 0.06),
0px 0px 0px 1px hsl(var(--shadow-color) / 0.04)",
  "0.94px 3.07px 2.89px -0.28px hsl(var(--shadow-color) / 0.08),
2.5px 8.19px 6.72px -0.65px hsl(var(--shadow-color) / 0.07),
4.88px 15.98px 12.07px -1.17px hsl(var(--shadow-color) / 0.07),
8.39px 27.47px 20.03px -1.94px hsl(var(--shadow-color) / 0.07),
13.61px 44.55px 33px -3.19px hsl(var(--shadow-color) / 0.06),
21.77px 71.3px 57.67px -5.58px hsl(var(--shadow-color) / 0.06),
35.27px 115.47px 124.74px -12.07px hsl(var(--shadow-color) / 0.06),
0px 0px 0px 1px hsl(var(--shadow-color) / 0.04)",
]
`;
